Sources/Private/Experimental/Layers/SolidLayer.swift

```diff
@@ -35,4 +34,14 @@ final class SolidLayer: BaseCompositionLayer {
 
   private let solidLayer: SolidLayerModel
 
+  private func setupContentLayer() {
+    // Render the fill color in a child `CAShapeLayer`
+    //  - Using a `CAShapeLayer` specifically, instead of a `CALayer` with a `backgroundColor`,
+    //    allows the size of the fill shape to be different from `contentsLayer.size`.
+    let shapeLayer = CAShapeLayer()
+    shapeLayer.fillColor = solidLayer.colorHex.cgColor
+    shapeLayer.path = CGPath(rect: .init(x: 0, y: 0, width: solidLayer.width, height: solidLayer.height), transform: nil)
```

Comment:
This is unrelated, but fixes this issue that's been bothering me:
<img width="755" alt="image" src="https://user-images.githubusercontent.com/1811727/148451823-f02aac05-b8bd-4e2c-855c-ce816fb1c3bb.png">